### DISCUSSION TITLE
ERD 테이블에 대한 피드백

### DISCUSSION THREAD

**Discussion by Aleexender** [SCBJ-7/SCBJ-BE#12] [Show and tell]
제 아는 현직자분한테 피드백을 요청드린결과 이러한 대답을 얻었어용!

양도 도메인인데 

숙박이 아니라 양도에 대한 확장 을 고려한다면?

주 도메인이 아니라 서브 도메인(숙박)에 너무 집중한 느낌

이게 주 도메인도 아닌데 굳이 알필요 까지 있나?
ex) 당근 마켓이 아이폰의 스팩까지 알려줄 필요가 있나??? → 이게 굳이 테이블로 들어가야하는지 고민해봐야한다.

제가 다시 봐본결과 ERD가 숙박업소에 대한 ERD인거같아요! 이거에 대해서 다시 얘기 해보면 좋을거같아요!

**Comment by Kwonminwoo** [comment]
좋습니다~~

**Comment by YurimYang** [comment]
오 좋습니다 !! 주 도메인에 대해 다시 잡고 서브 도메인을 줄이는 작업을 해봅시당 💯
